musicbrainz-server/templates/default/sv-musicbrainz-server-run.erb

```diff
@@ -5,10 +5,14 @@ set -e
 cd mb_server
 umask 022
 exec \
-    env MUSICBRAINZ_USE_PROXY=1 starman -Ilib \
-        --preload-app \
-        -E deployment \
-        --listen musicbrainz-server.socket \
-        --workers <%= @variables[:nproc] || 5 %> \
-        --user www-data \
-        --group www-data
+    env MUSICBRAINZ_USE_PROXY=1 \
+        setuidgid www-data \
+            start_server --path musicbrainz-server.socket -- \
+                plackup \
+                    -Ilib \
+                    --server Starlet \
+                    --env deployment \
+                    --max-workers <%= @variables[:nproc] || 5 %> \
+                    --min-reqs-per-child 100 \
+                    --max-reqs-per-child 1000 \
```

Comment:
The default `max-reqs-per-child` is 100, but our current equivalent Starman setting is 1000, so I left it like that. I did set `min-reqs-per-child` to 100 though, which is a new setting. It causes Starlet to pick a random value between the two for each worker. The idea with that, as I understand it, is to not have every worker die/need re-forking at around the exact same time.